src/main/java/com/lizin5ths/indypets/util/IndyPetsUtil.java

```diff
@@ -56,13 +56,13 @@ public static boolean isActive(Entity entity) {
 		if (!tameable.isTamed())
 			return false;
 
-		Config config = ServerConfig.getDefaultedPlayerConfig(tameable.getOwnerUuid());
+		Config config = ServerConfig.getDefaultedPlayerConfig(tameable.getOwnerReference().getUuid());
```

Comment:
`getOwnerReference()` is nullable and `.getUuid()` could theoretically throw a NPE, which realistically shouldn't happen here, but it's not as clear in the second instance.

When the reference is `null`, it should pass `null` to `getDefaultedPlayerConfig()`, so it'll use the default config, for example like this:
```
var ownerRef = tameable.getOwnerReference().getUuid();
ServerConfig.getDefaultedPlayerConfig(ownerRef != null ? ownerRef.getUuid() : null)
```

Since that code's ugly and used twice, I'd put this in an overload, so you can just write
`ServerConfig.getDefaultedPlayerConfig(tameable.getOwnerReference());`